content/project-news/reactos-in-2020.md

```diff
@@ -0,0 +1,72 @@
+---
+title:       "ReactOS in 2020"
+author:      "Extravert34"
+date:        2020-12-28
+---
+
+Despite all the turbulence, we've got quite a rich year for ReactOS.
+Many bugs and instabilities were resolved, many more have been introduced. This year we hired two kernel developers full-time, this happens for the first time on the project history.
+
+The post highlights some of the changes which may may be interesting to the community.
+
+## Shell changes
+
+Shell doesn't see much attention recently, due to most of the work concentrated in the kernel, but there are still some useful fixes and feature implementations.
+
+* Katayama Hirofumi MZ made a great job implementing the Filesystem change notification feature (aka `SHChangeNotify`) in the shell. This is required for all applications monitoring the filesystem changes
+* Visual and usability improvements: "Size on disk" label in a file Properties dialog, key accelerators in Open/Save dialog and many more
+
+## RAPPS (a GSoC project)
+
+This is one of our three GSoC projects of the year. He Yang continues the work from previous GSoC years, by improving our RAPPS application manager. This includes UI & usability enhancements, showing app screenshots, command line scripting support.
+
+*picture*
+
+## Compiler and tooling upgrade
+
+The aging RosBE 2.1 toolset has served us well since 2014, but it's time to move on and incorporate all the useful features implemented in modern compilers and build tools.
+We now have GCC 8.4, CMake 3.17 as our defaults in the new RosBE 2.2, while maintaining compatibility with the latest CMake. The transition didn't happen without flaws though, a lot of effort was put by Colin Finck to make modern CMake work on Windows 2003 (and thus on ReactOS because we maintain compatibility with it). All the patches are freely available in our GitHub.
```

Comment:
```suggestion
We now have GCC 8.4 and CMake 3.17 as our defaults in the new RosBE 2.2, while maintaining compatibility with the latest CMake.
The transition didn't happen without a few road bumps though, a lot of effort was put by Colin Finck to make modern CMake work on Windows Server 2003 (and thus on ReactOS because we maintain compatibility with it).
All the patches are freely available in our [GitHub](https://github.com/reactos/CMake/tree/cmake-3.17.2-reactos).
```